src/raft_server.cpp

```diff
@@ -499,6 +500,14 @@ int ReplicationState::init_db() {
         return 1;
     }
 
+    RateLimitManager *rateLimitManager = RateLimitManager::getInstance();
```

Comment:
Since we are using the `meta_store` now, it's okay to init the rate limit manager in the earlier place. We should not do it here.